packages/wallet-adapter-core/src/LegacyWalletPlugins/types.ts

```diff
@@ -96,32 +108,48 @@ export interface AdapterPluginProps<Name extends string = string> {
   name: WalletName<Name>;
   url: string;
   icon: `data:image/${"svg+xml" | "webp" | "png" | "gif"};base64,${string}`;
-  version?: "v1" | "v2";
   providerName?: string;
   provider: any;
+  // Compatible with legacy wallet plugin
   deeplinkProvider?: (data: { url: string }) => string;
+  // Comaptible with AIP-62 standard wallet
+  openInMobileApp?: () => void;
   connect(): Promise<any>;
   disconnect: () => Promise<any>;
   network: () => Promise<any>;
-  signAndSubmitTransaction(
-    transaction: Types.TransactionPayload | InputGenerateTransactionData,
+  signAndSubmitTransaction?(
+    transaction:
+      | Types.TransactionPayload
```

Comment:
Kinda weird, didn't know you could put a `|` before and have nothing before it

Reply:
it is just a format thing, we are having it across other ts tools